test/core.js

```diff
@@ -615,6 +619,16 @@ describe('snabbdom', function() {
           assert.deepEqual(map(inner, elm.children), arr.filter(function(x) {return x != null;}));
         }
       });
+      it('accepts symbol as key', function() {
+        var vnode1 = h('span', [Symbol()].map(spanNum));
+        var vnode2 = h('span', [Symbol('1'), Symbol('2'), Symbol('3')].map(spanNum));
+        elm = patch(vnode0, vnode1).elm;
+        assert.equal(elm.children.length, 1);
```

Comment:
I'm sorry. I'm the culprit. I have guided @guri through this PR. I was certain that this is about having symbols as `key`s. @aronallen, what did you mean in #124, please?